### PR TITLE
Feature - Client cleanup

### PR DESCRIPTION
**Pull request by Rican7**
This does some cleanup of the clients to make it a bit easier to implement client class extensions.

### CODE REVIEW

File: NTLMSoapClient.php [hunk 1]
```diff
@@ -44,6 +44,20 @@ class NTLMSoapClient extends SoapClient
     protected $validate = false;
 
     /**
+     * Username for authentication on the exchnage server
```
Comment (jstruzik) [inline NTLMSoapClient.php]: spelling

**Comment by jstruzik** [comment]
:+1:

File: NTLMSoapClient.php (hunk 1 above)
Comment (Rican7) [inline NTLMSoapClient.php]: Whoops! I had just moved it from the other file. But yea, I'll fix up.

**Comment by jstruzik** [comment]
GTG from me still :+1:

**Comment by brianmuse** [comment]
Yeah this LGTM :+1:

**Comment by Rican7** [comment]
Thanks!

**Pull request merged**